modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -3352,12 +3376,16 @@ label:deprecated[]
 CREATE DATABASE databaseName.withDot ...
 ----
 a|
-Creating a database with dots in the name has been deprecated, instead quote the database name using backticks:
+Creating a database with dots in the name has been deprecated in versions 5.26 to 2025.05.
+However, starting from version 2025.06, this deprecation has been withdrawn, and replaced by two new deprecations.
+The reason for this change is that the previous quoting rules have been found inconsistent and ambiguous, and will be improved starting from Cypher 25.
+For more information, see:
 
-[source, cypher, role="noheader"]
-----
-CREATE DATABASE `databaseName.withDot` ...
-----
+* xref::deprecations-additions-removals-compatibility.adoc#_graph_reference_removed_name_parts_quoting[removed support for quoted name parts in graph references]
+
+* xref::deprecations-additions-removals-compatibility.adoc#_graph_reference_updated_graph_by_name_quotes[update to graph functions]
+
+* xref::values-and-types/graph-references.adoc#rules[graph reference rules]
```

Comment:
Creating a database with `.` characters was deprecated in 5.0.
However, this deprecation was withdrawn in 2025.06, and replaced by the following updates:

* xref::deprecations-additions-removals-compatibility.adoc#_graph_reference_removed_name_parts_quoting[Removed support for quoted name parts in graph references]
* xref::deprecations-additions-removals-compatibility.adoc#_graph_reference_updated_graph_by_name_quotes[Update to graph functions]

This because the previous quoting rules were inconsistent and ambiguous.
The rules are clarified and improved in Cypher 25.
For more information, see:

* xref::values-and-types/graph-references.adoc#rules[Graph reference rules]